File: 3856-trim-trailing-vowels/3856-trim-trailing-vowels.java
class Solution {
    public String trimTrailingVowels(String s) {
        String vowels = "AEIOUaeiou";

        int i = s.length()-1;

        while(i >= 0){
            if(vowels.indexOf(s.charAt(i)) != -1){
                i--;
            }else
                break;
        }

        return s.substring(0, i+1);
    }
}

File: 3856-trim-trailing-vowels/3856-trim-trailing-vowels.cpp
class Solution {
public:
    string trimTrailingVowels(string s) {

        string vowels = "aeiouAEIOU";

        for(int i=s.length()-1;i>=0;i--){
            if(vowels.contains(s[i])){
                s.pop_back();
            }
            else{
                break;
            }
        }

        return s;

    }
};